sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/SkipLocalsInitAttribute.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.Runtime.CompilerServices
```

Comment:
Do we need to wrap this in an `#ifdef` or is it just a marker interface that would have the same effect as the actual system implementation?